packages/api/src/logger/index.ts

```diff
@@ -244,19 +242,24 @@ export interface RedwoodLoggerOptions {
 export const createLogger = ({
   options,
   destination,
-  showConfig = false,
-}: RedwoodLoggerOptions): BaseLogger => {
+  showConfig = true, // change to false
+}: RedwoodLoggerOptions): P.BaseLogger => {
```

Comment:
Would an option (and open for discussion if good or bad) be to create Redwood types like

`RedwoodBaseLogger` or to that effect that is simply the `pino.BaseLogger` or `P.BaseLogger` so that in the user's app there is less Pino and more Redwood seen?

Reply:
Since we don't import Pino types inside Redwood App, I don't see the necessity, but it can't be bad except maybe for the maintenance to keep these type up to date with Pino.